assets/component-cart.css

```diff
@@ -87,19 +87,26 @@ cart-items {
 
 .cart__note {
   display: block;
+  height: 100%;
 }
```

Comment:
When trying to type in a message in the cart note the styles are a bit off: 
![](https://screenshot.click/22-01-0gody-4mga9.png)